dev/doc/build-system.dune.md

```diff
@@ -112,9 +112,13 @@ ml files in quick mode.
 Dune also provides targets for documentation, testing, and release
 builds, please see below.
 
-## Documentation and testing targets
+## Testing and documentation targets
 
-Coq's test-suite can be run with `dune runtest`; given that `dune`
+For the moment, to run the test-suite, we recommend calling `make`
+directly from the test-suite directory (or with `make -C test-suite`).
+This is the only way to get incremental test-suite builds.
```

Comment:
Perhaps a comment about there being documentation on the targets inside that actual makefile. i.e. "See test-suite/Makefile for more info" or something.

Reply:
Currently there is doc spread between the Makefile and the README in the test-suite so maybe both should be mentioned?